src/main/kotlin/lotto/Rank.kt

```diff
@@ -0,0 +1,21 @@
+package lotto
+
+import java.util.function.BiPredicate
+
+enum class Rank(val isMatched: BiPredicate<Int, Boolean>, prize: Int) {
+    FIRST({ count, _ -> count == 6 }, 2000000000),
```

Comment:
꿀팁을 하나 드리자면 이렇게 큰 수를 기재할 때 아래와 같이 표현할 수 있습니다.

```suggestion
    FIRST({ count, _ -> count == 6 }, 2_000_000_000),
```